paymentsheet/src/main/java/com/stripe/android/common/ui/PrimaryButton.kt

```diff
@@ -91,8 +97,13 @@ private fun PrimaryButtonContent(
     text: String,
     color: Color,
     style: TextStyle,
+    isEnabled: Boolean,
     isLoading: Boolean,
+    displayLockIcon: Boolean,
 ) {
+    val context = LocalContext.current
```

Comment:
Can we move these into the if block, since they're not needed outside of it?

Reply:
I will fix this up in a follow up PR!